BonApp.js

```diff
@@ -3,11 +3,9 @@ refreshFrequency: 300000,
 
 render: function(output) {
   return [
-      '<link rel="stylesheet" href="BonApp-Widget/styles/jquery/jquery-ui.css">',
       '<link rel="stylesheet" href="BonApp-Widget/styles/bonAppStyle.css">',
-
-      '<script src="BonApp-widget/scripts/jquery/jquery-1.10.2.js"></script>',
-      '<script src="BonApp-widget/scripts/jquery/jquery-ui.js"></script>',
+      '<link rel="stylesheet" href="BonApp-widget/styles/jquery/jquery-ui.css">',
```

Comment:
`-widget` needs to be capitalized here

Reply:
Maybe jquery-ui isn't necessary if the widget works without this path being correct.